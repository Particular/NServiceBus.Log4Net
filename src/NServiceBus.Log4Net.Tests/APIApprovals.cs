using System.IO;
using ApiApprover;
using NServiceBus;
using NUnit.Framework;

[TestFixture]
public class APIApprovals
{
    [Test]
    public void Approve()
    {
        Directory.SetCurrentDirectory(TestContext.CurrentContext.TestDirectory);
        PublicApiApprover.ApprovePublicApi(typeof(Log4NetFactory).Assembly);
    }

}