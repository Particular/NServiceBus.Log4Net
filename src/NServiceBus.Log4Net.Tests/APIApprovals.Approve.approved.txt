[assembly: System.CLSCompliantAttribute(true)]
[assembly: System.Runtime.Versioning.TargetFrameworkAttribute(".NETFramework,Version=v4.5.2", FrameworkDisplayName=".NET Framework 4.5.2")]
namespace NServiceBus
{
    public class Log4NetFactory : NServiceBus.Logging.LoggingFactoryDefinition
    {
        public Log4NetFactory() { }
        protected override NServiceBus.Logging.ILoggerFactory GetLoggingFactory() { }
    }
}